pkg/services/repo_service.go

```diff
@@ -28,8 +29,16 @@ type argoCDService struct {
 }
 
 type Repos interface {
+	// GetApps return a list of valid Argo CD Application sources within the repo, as per the rules described here: https://argoproj.github.io/argo-cd/user-guide/tool_detection/
 	GetApps(ctx context.Context, repoURL string, revision string) ([]string, error)
```

Comment:
Is there any other use case of GetApps? If not, we can delete it.

Reply:
Agreed @OmerKahani , and done.